templates/components/common/share.html

```diff
@@ -61,11 +61,13 @@
             {{/each}}
         </ul>
         <script type="text/javascript"
-                src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4e94ed470ee51e32"></script>
-        <script>
-            if (typeof(addthis) === "object") {
-                addthis.toolbox('.addthis_toolbox');
-            }
+                defer src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4e94ed470ee51e32"></script>
+        <script>  
+            window.onload = function() {
```

Comment:
This is not a good approach because it will wipe out any other onload handlers that were created. You should just jquery to attach an on load handler instead.

Reply:
Actually, turns out jQuery is not available at this time, because we load it as part of the bundle which is inserted at the end of the body (after this). So we need to use vanilla javascript here. `document.addEventListener('DOMContentLoaded', handler);`